cypress/support/pages/signIn.pageObject.js

```diff
@@ -5,30 +5,25 @@ class SignInPageObject extends PageObject {
 
   get emailField() {
     return cy.getByDataCy('email-sign-in');
+    return this.getByDataCy('email-sign-in');
```

Comment:
There is a duplicate return statement in the `emailField` getter. Remove the second `return this.getByDataCy('email-sign-in');` to avoid redundancy and potential errors.